libcxx/test/std/ranges/range.factories/range.repeat.view/views_repeat.pass.cpp

```diff
@@ -60,6 +60,14 @@ static_assert(!std::is_invocable_v<decltype(std::views::repeat), NonCopyable>);
 // Tp is move_constructible
 static_assert(std::is_invocable_v<decltype(std::views::repeat), MoveOnly>);
 
+// LWG4053: Unary call to std::views::repeat does not decay the argument
+using RPV = std::ranges::repeat_view<const char*>;
+
+static_assert(std::same_as<decltype(std::views::repeat("foo", std::unreachable_sentinel)), RPV>);  // OK
+static_assert(std::same_as<decltype(std::views::repeat(+"foo", std::unreachable_sentinel)), RPV>); // OK
+static_assert(std::same_as<decltype(std::views::repeat("foo")), RPV>);                             // OK since LWG4053
+static_assert(std::same_as<decltype(std::views::repeat(+"foo")), RPV>);                            // OK
```

Comment:
These cases are actually covered by the resolution of LWG4054. As both LWG4053 and LWG4054 are accepted, the resolution of LWG4053 only affects CTAD.

I think these lines should be moved to PR #88606, and `ctad.compile.pass.cpp` should be modified in this PR.

Reply:
Thanks for your tips! Seems LWG4053 and LWG4054 should swap proposed resolutions.

Reply:
Does the follow case also well-formed?
```
static_assert(std::same_as<decltype(std::ranges::repeat_view(std::ranges::repeat_view(1))), std::ranges::repeat_view<std::ranges::repeat_view<int>>>);
```

Reply:
> Does the follow case also well-formed?
> 
> ```
> static_assert(std::same_as<decltype(std::ranges::repeat_view(std::ranges::repeat_view(1))), std::ranges::repeat_view<std::ranges::repeat_view<int>>>);
> ```

No. The copy deduction candidate is selected here, so the deduced type is `std::ranges::repeat_view<int>`.
This is an unavoidable divergence between `views::repeat` and CTAD for `repeat_view`, and user code should always use `views::repeat`.

Reply:
Thank you for your confirmation!